CHANGELOG.md

```diff
@@ -174,207 +188,37 @@
 
 <a name="v0.4.4"></a>
 ## [v0.4.4] - 2022-01-20
-### Bug Fixes
```

Comment:
should these messages from previous releases not be added?

Reply:
ouch, this is the changelog generator messing around... What should we do? The changelog should not change over the time... is there a way to tell it to ignore these old things or should we just remove them once, to avoid this issue from happening again?

Reply:
I vote for only staging the last version, and the bottom of the file with the compare links.

Reply:
I cleaned up the changelog to ensure only new data is added, old parts of the file are not changed